test/core/messages.integration.test.ts

```diff
@@ -255,24 +267,32 @@ describe('messages integration', () => {
     const deletedMessage1 = await room.messages.delete(message1, { description: 'Deleted message' });
 
     // Do a history request to get the deleted message
+    await new Promise((resolve) => setTimeout(resolve, 3000)); // wait for persistence - this will not be necessary in the future
     const history = await room.messages.get({ limit: 3, direction: 'forwards' });
 
     expect(history.items).toEqual([
       expect.objectContaining({
         text: 'Hello there!',
         clientId: chat.clientId,
         serial: deletedMessage1.serial,
-        deletedAt: deletedMessage1.deletedAt,
-        deletedBy: chat.clientId,
+        createdAt: message1.timestamp,
+        timestamp: deletedMessage1.deletedAt,
+        action: ChatMessageActions.MessageDelete,
       }),
     ]);
 
+    // test shorthand getters
+    expect(history.items[0]?.isUpdated).toBe(false);
+    expect(history.items[0]?.isDeleted).toBe(true);
+    expect(history.items[0]?.deletedAt).toEqual(deletedMessage1.deletedAt);
+    // expect(history.items[0]?.deletedBy).toEqual(deletedMessage1.deletedBy);
```

Comment:
Should this be un-commented?

Reply:
I'm adding a PR to fix this in realtime first (nearly done), I don't think we should wait for this PR though